File: github_cpp/21/100.cpp
#include<iostream>
using namespace std;

int main(){

    int n;
    cout<<"Enter N";
    cin>>n;

    int a[100];
    
    for(int i=0;i<n;i++){
        cin>>a[i];
    }
    
    for(int i=0;i<n-1;i++){

        int min_i = i;
        
        for(int j=i+1;j<=n-1;j++){
            if(a[j]<a[min_i]){
                min_i = j;
            }
        }
        
        int temp = a[min_i];
        a[min_i] = a[i];
        a[i] = temp;
    }
    
    for(int i=0;i<n;i++){
        cout<<a[i]<<" ";
    }





}

File: github_code/tower-of-hanoi/java/100.java
import java.util.Scanner;

/**
 This program is used for solving tower of hanoi.

*/
public class towerofhanoi {
	/**
		Recursive method for solving the program
		@Param:
			n: int, number of disc
			start: the Starting Pole
			temp :place to  storing disk temporarily
			end: end: final spot

	*/
	   public void solve(int n, String start, String temp, String end) {
	       if (n == 1) {
	           System.out.println(start + " -> " + end);
	       } else {
	           solve(n - 1, start, end, temp);
	           System.out.println(start + " -> " + end);
	           solve(n - 1, temp, start, end);
	       }
	   }
		
	// Main method to input number of disc.
	   public static void main(String[] args) {
	       towerofhanoi towersOfHanoi = new towerofhanoi();
	       System.out.print("Enter number of discs: ");
	       Scanner scanner = new Scanner(System.in);
	       int discs = scanner.nextInt();
	       towersOfHanoi.solve(discs, "A", "B", "C");
	   }
	}
